ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -138,3 +138,31 @@ def test_numpy_fftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device)
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.fft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2, 2),
+    ),
+    s=st.sampled_from([(2, 2), (3, 3), (4, 4)]),
```

Comment:
I'm not that familiar with FFT but are those the only allowed values? What if I use `s=(5, 5)`?

Reply:
that's right, will change to an st.integers() instead if that is better suited to this